store/principal.go

```diff
@@ -78,7 +85,9 @@ func (s *Store) GetPrincipalList(ctx context.Context) ([]*api.Principal, error)
 		if err != nil {
 			return nil, errors.Wrapf(err, "failed to compose Principal role with principalRaw[%+v]", raw)
 		}
-		principalList = append(principalList, principal)
+		if principal != nil {
```

Comment:
Seems like a bug here, the principal and err can both be nil https://github.com/bytebase/bytebase/blob/main/store/member.go#L240
So we should check if the principal is not nil

Reply:
True, let's do this in a separate PR